### ISSUE TITLE
miniframe navigation without using subsections

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#172]
Dear Joseph,

I have a small question about the miniframe outer theme. Unfortunately these little bullets for each slide are only drawn, if there is at least one subsection in every section of the presentation.

So also in the following minimal example:

```latex
\documentclass{beamer}
\usepackage[ngerman]{babel} \usepackage{lmodern}
\useoutertheme{miniframes}
\begin{document}
\section{erster Abschnitt}

\subsection*{}

\begin{frame}{erste Folie} Inhalt \end{frame}

\begin{frame}{zweite Folie} Inhalt \end{frame}

\end{document}
```

the bullets are correctly drawn. But if the subsection-command is removed, the bullets disappear.

Is this a wanted behaviour of the beamer package that every section has to have at least one (even empty) subsection for the bullets to appear?

This behaviour is also not documented in the `beamer` user guide.

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/172
- Originally reported by: Anonymous 
- Originally created at:  2012-01-20

**Comment by josephwright** [comment]
_From Joseph Wright on 2012-02-22_

You need `\useoutertheme[subsection = false]{miniframes}`: I will look to see where this should be in the documentation!

**Comment by josephwright** [comment]
Fixed by 6d3df67ba1719fe2816f361297ffd078883fca53.
